lib/api/dsl/methods.js

```diff
@@ -7,8 +7,202 @@ var
   q = require('q'),
   util = require('util'),
   geohash = require('ngeohash'),
+  units = require('node-units'),
```

Comment:
yep but since the units conversions dont use to move that much... is it really important ?